### PR TITLE
Add an end-to-end test for managed dependencies

### PR DESCRIPTION
**Pull request by AnatoliB**
Add an end-to-end test for managed dependencies.

### CODE REVIEW

**Comment by AnatoliB** [comment]
I'm planning to merge this _before_ https://github.com/Azure/azure-functions-powershell-worker/pull/267, so that we establish the baseline first.

File: test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/ManagedDependenciesEndToEndTests.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        [Fact]
+        public async Task ManagedDependenciesAreAvailable()
+        {
+            var actualResponseMessage =
+                await Utilities.InvokeHttpTrigger("UsingManagedDependencies", string.Empty, HttpStatusCode.OK);
+            Assert.Contains(@"\ManagedDependencies\Az\", actualResponseMessage);
+        }
+    }
+}
```
Comment (Francisco-Gamino) [inline test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/ManagedDependenciesEndToEndTests.cs]: Could you please add new line?

Comment (AnatoliB) [reply test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/ManagedDependenciesEndToEndTests.cs]: Sure :-)

File: test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/Utilities.cs [hunk 2]
```diff
@@ -53,5 +47,27 @@ public static async Task<bool> InvokeHttpTrigger(string functionName, string que
...
+            return await response.Content.ReadAsStringAsync();
+        }
+
+        private static async Task<HttpResponseMessage> GetHttpTriggerResponse(string functionName, string queryString)
+        {
+            string uri = $"api/{functionName}{queryString}";
+            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, uri);
+            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("text/plain"));
+
+            var httpClient = new HttpClient();
```
Comment (TylerLeonhardt) [inline test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/Utilities.cs]: Pretty sure we could reuse an HttpClient that way we’re not allocating one per-request

Comment (AnatoliB) [reply test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/Utilities.cs]: We definitely could, but I'm not convinced the benefits (saving a few milliseconds per test run?) outweigh the drawbacks (increased coupling between tests). Unless you find a really good reason, I will leave it as is.

Comment (TylerLeonhardt) [reply test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/Utilities.cs]: That's fine, just wanted to point it out.

Comment (Francisco-Gamino) [reply test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/Utilities.cs]: Please consider using the ```using statement``` for the httpclient. Other than that, LGTM.

Comment (AnatoliB) [reply test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/Utilities.cs]: > Please consider using the `using statement` for the httpclient. Other than that, LGTM.

Thank you, let's do it next time. This PR does not make anything worse: this is not new code, I'm just extracting a method.

Comment (Francisco-Gamino) [reply test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/Utilities.cs]: Sounds good. Thanks @AnatoliB

**Pull request merged**